Week-1/cattyboard/styles.css

```diff
@@ -0,0 +1,140 @@
+* {
+    font-family: 'Lato', sans-serif;
+    margin: 0;
+    padding: 0;
+}
+
+nav{
+    min-width: 4%;
```

Comment:
4% ? How did you come around this value ?

Reply:
I've experimented with various values and found 4% closely matching with the required design.
So, used 4% (to make it responsive).